plugins/servicenow/help.md

```diff
@@ -103,11 +103,7 @@ Example input:
 
 ```
 {
-  "create_data": {
-    "Description": "Bug report",
-    "ID": "58",
-    "date": "2021-08-20 18:12:00"
-  },
+  "create_data": "{\"Description\": \"Bug report\", \"ID\": \"58\", \"date\": \"2021-08-20 18:12:00\"}",
```

Comment:
Is this to pass validation? Example will be displayed as a string and not object.